tool.go

```diff
@@ -921,22 +921,8 @@ var examples = []testing.InternalExample{
 {{end}}
 }
 
-var matchPat string
-var matchRe *regexp.Regexp
-
-func matchString(pat, str string) (result bool, err error) {
-	if matchRe == nil || matchPat != pat {
-		matchPat = pat
-		matchRe, err = regexp.Compile(matchPat)
-		if err != nil {
-			return
-		}
-	}
-	return matchRe.MatchString(str), nil
-}
-
 func main() {
-	m := testing.MainStart(matchString, tests, benchmarks, examples)
+	m := testing.MainStart(testdeps.TestDeps{}, tests, benchmarks, examples)
```

Comment:
The commit message of 3b29828ba7778097142fd8db96ecf75d9d3a341c says:

> See https://github.com/golang/go/commit/c56cc9b3b5727647c2afb3d57f5793151558a0a7?w=1.

That commit shows that it's still implemented the same way, using `regexp`. They've just shuffled imports around into `testdeps` package.